src/common/Navbar/index.js

```diff
@@ -43,6 +46,14 @@ const Navigation = () => {
                 People
               </StyledNavLink>
             </NavigationListItem>
+            <NavigationListItem>
+              <StyledNavLink favorite to={toFavoriteMovies()}>
```

Comment:
I noticed that console show waring with property favorite.

 Please see below:
![image](https://user-images.githubusercontent.com/67117780/97778841-528f1180-1b7a-11eb-81f6-387d14420633.png)

If you change to favorite={"favorite") warning disappear.